apps/deploy-web/src/components/billing-usage/BillingContainer/BillingContainer.tsx

```diff
@@ -45,9 +48,14 @@ export const BillingContainer: React.FC<BillingContainerProps> = ({ children, de
 
   const [cursorHistory, setCursorHistory] = useState<CursorHistoryItem[]>([{ pageIndex: 0 }]);
 
-  const [error, setError] = React.useState<Error | null>(null);
+  const [errorMessage, setErrorMessage] = React.useState("");
 
-  const { from, to } = dateRange;
+  const { from: startDate, to: endDate } = dateRange;
+
+  const exportCsvQuery = D.useExportTransactionsCsvQuery({
+    startDate,
+    endDate
+  });
```

Comment:
question(non-blocking): do we even need to make it via a hook? This could be done via service call on button.